src/Features/CSharp/Portable/InitializeParameter/InitializeParameterHelpers.cs

```diff
@@ -19,18 +19,30 @@ public static bool IsFunctionDeclaration(SyntaxNode node)
             || node is LocalFunctionStatementSyntax
             || node is AnonymousFunctionExpressionSyntax;
 
-        public static IBlockOperation GetBlockOperation(SyntaxNode functionDeclaration, SemanticModel semanticModel, CancellationToken cancellationToken)
+        public static bool TryGetBlockOperation(SyntaxNode functionDeclaration, SemanticModel semanticModel, CancellationToken cancellationToken, out IBlockOperation blockOperation)
         {
             var bodyOpt = GetBody(functionDeclaration);
             if (bodyOpt == null)
             {
-                return null;
+                blockOperation = null;
+                return true;
```

Comment:
"TryX" is returning null *and* true?  that's really inconsistent with all other uses of this pattern.

Reply:
I agree this is very confusing.